python/static-site/README.md

```diff
@@ -0,0 +1,58 @@
+# Static Site with AWS CDK
+This sample project uses S3, CloudFront and Route53 to create a stack for 
+hosting a static web site.
+
+## Prerequisites
+- A registered domain in Route53.
+- If using a public S3 bucket, a secret phrase needs to be created in the parameter store. The name of the parameter to be passed in the context as `origin_custom_header_parameter_name`. Make sure to create the parameter in the same region as the static site. This parameter will be set in both CloudFront and S3 Bucket Policy to make sure only traffic from CloudFront can access the S3 objects.
+- If you're using an existing certificate, make sure it's in the `us-east-1` region otherwise the deployment fails. See this https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/cnames-and-https-requirements.html#https-requirements-aws-region
+- The AWS environment needs to be bootstrapped.
+```sh
+cdk bootstrap aws://123456789012/us-east-1
+```
```

Comment:
Would you mind adding the standard python setup/build/deploy/cleanup instructions? You should be able to find those in other python example READMEs.

Reply:
Done! Please review.